server/lib/queries.js

```diff
@@ -1137,8 +1146,8 @@ const getTransactionsTimeSeries = async (kind, type, hostCollectiveId, timeUnit,
          AND type = :type
          AND "deletedAt" IS NULL
          ${collectiveIds ? `AND "CollectiveId" IN (:collectiveIds)` : ``}
-         ${dateFrom ? `AND "createdAt" >= :dateFrom` : ``}
-         ${dateTo ? `AND "createdAt" <= :dateTo` : ``}
+         ${startDate ? `AND "createdAt" >= :startDate` : ``}
+         ${endDate ? `AND "createdAt" <= :endDate` : ``}
```

Comment:
It's still a problematic and confusing pattern that the `startDate` used in the condition is not the same as the `startDate` passed as replacement. 

It's making the code really difficult to follow

Reply:
This should be refactored.

Reply:
What's the point of computing a default value for `startDate`/`endDate` in `computeDates` if we're not using it?

Reply:
Or, if you're not interested in the default values, then just don't use `computeDates`.

Reply:
Yes, I've just refactored it so that we remove this default value condition. I've renamed `computeDates` to `computeDatesAsISOStrings` since that's what this function does currently. Let me know if any other refactoring needs to be done.